feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/SessionsItemDecoration.kt

```diff
@@ -19,11 +19,13 @@ import io.github.droidkaigi.confsched2019.session.R
 import io.github.droidkaigi.confsched2019.session.ui.item.SessionItem
 import io.github.droidkaigi.confsched2019.timber.debug
 import timber.log.Timber
+import java.lang.ref.WeakReference
 
 class SessionsItemDecoration(
     val context: Context,
-    val groupAdapter: GroupAdapter<*>
+    groupAdapter: GroupAdapter<*>
```

Comment:
It seems this class wants only item count of an adapter, and the direction is restricted to vertical.
So we don't need passing an adapter to this class, and then we can use `RecyclerView#getRowCountForAccessibility` or `RecyclerView#getAdapter()?.getItemCount()` instead.

Reply:
BTW, `RecyclerView$State#getItemCount` cannot be used in this case because it looks up the prelayout state.

Reply:
Unfortunately, this class uses `GroupAdapter#getItem` in the `getSessionTime` method 😇

Reply:
Oh sorry, I was wrong. 😞